lib/editing/editor_comid.dart

```diff
@@ -73,10 +69,7 @@ class ComidFactory extends EditorFactory {
         'indentUnit': 2,
         'cursorHeight': 0.85,
         //'gutters': [_gutterId],
-        'extraKeys': {
-          'Ctrl-Space': 'autocomplete',
-          (mac ? "Cmd-/" : "Ctrl-/"): "toggleComment"
-        },
+        'extraKeys': { (mac ? "Cmd-/" : "Ctrl-/"): "toggleComment" },
```

Comment:
It is handled elsewhere by dart pad. Including it here caused a cancellation exception.